Celeste.Mod.mm/Mod/Everest/EverestSplash.cs

```diff
@@ -0,0 +1,578 @@
+﻿#nullable enable
+using System;
+using System.IO;
+using System.IO.Pipes;
+using SDL2;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+using System.Timers;
+
+namespace EverestSplash;
```

Comment:
should be in namespace `Celeste.Mod` like the other stuff in this folder

Reply:
Whoops!